test/config/AppConfigSpec.scala

```diff
@@ -150,11 +150,11 @@ class AppConfigSpec extends CustomExportsBaseSpec {
     }
 
     "have countryCodesJsonFilename" in {
-      validConfigService.countryCodesJsonFilename must be("location-autocomplete-canonical-list.json")
+      validConfigService.countryCodesJsonFilename must be("code-lists/location-autocomplete-canonical-list.json")
```

Comment:
why are we having json source and a csv source for countries -

Reply:
Beyond the scope of this story, we can raise a tech debt ticket if you like

Reply:
I agree with @ed0906 but moreove our countries are in json file so we should stay with json, we have support for csv and now when we add new stuff and we receive new stuff in csv we can use our new approach, but IMO we should support both things, read from Json and from csv, I'm not sure that we should convert all countries to csv, we're using this correctly.